src/index.ts

```diff
@@ -1,36 +1,34 @@
 let vacias: number = 0;
 let ocupadas: number = 0;
-let indice: number;
-
```

Comment:
Este indice lo borramos, y lo declaramos dentro de cada for.
Poner el indice del for como una variable global no es una buena practica.